examples/coexist_klog_v1_using_logr/go.mod

```diff
@@ -0,0 +1,10 @@
+module k8s.io/klog/examples/coexist_klog_v1
+
+go 1.13
+
+require (
+	k8s.io/klog v1.0.0
+	k8s.io/klog/v2 v2.0.0
```

Comment:
v2.0.0 won't have the depth constants, will need to tag a v2.0.1 or something (applies everywhere)